app/models/katello/content_view_erratum_filter.rb

```diff
@@ -23,12 +23,11 @@ def generate_clauses(_repo)
         start_date = erratum_rules.first.start_date
         end_date = erratum_rules.first.end_date
         types = erratum_rules.first.types
-
         unless start_date.blank? && end_date.blank?
           date_range = {}
           date_range["$gte"] = start_date.to_time.as_json unless start_date.blank?
           date_range["$lte"] = end_date.to_time.as_json unless end_date.blank?
-          rule_clauses << { "issued" => date_range }
+          rule_clauses << { erratum_rules.first.date_type => date_range }
```

Comment:
I could see some opportunity for future breakage by tying the 'date_type' in our db to the 'date field' in mongo.  Would you mind separating these two?   As a developer i may not realize its using this db field for the mongo queries